src/check-reserve-update-frequency/check-reserve-update-frequency.js

```diff
@@ -0,0 +1,166 @@
+const ethers = require('ethers');
+const BigNumber = require('bignumber.js');
+const {
+  Finding, FindingSeverity, FindingType, getJsonRpcUrl,
+} = require('forta-agent');
+
+// load required shared types
+const {
+  LendingPoolAddressesProvider: lendingPoolAddressesProvider,
+  ProtocolDataProvider: protocolDataProviderAddress,
+} = require('../../contract-addresses.json');
+const { abi: protocolDataProviderAbi } = require('../../interfaces/AaveProtocolDataProvider.json');
+const { abi: priceOracleAbi } = require('../../interfaces/IPriceOracle.json');
+const { abi: chainlinkAggregatorAbi } = require('../../interfaces/IChainlinkAggregator.json');
+const { abi: lendingPoolAddressesProviderAbi } = require('../../interfaces/ILendingPoolAddressesProvider.json');
+
+// set up the an ethers provider
+const jsonRpcProvider = new ethers.providers.WebSocketProvider(getJsonRpcUrl());
```

Comment:
this needs to change from `WebSocketProvider` to `getDefaultProvider`

Reply:
Done.